jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt

```diff
@@ -177,11 +178,11 @@ class LogStreamDownloadToFileTask(
                     }
                 )
             )
-            CloudwatchlogsTelemetry.downloadStreamToFile(project, success = true)
+            CloudwatchlogsTelemetry.download(project, Result.Succeeded, CloudWatchResourceType.LogStream)
```

Comment:
Probably need to ensure parity while recording - result = Result.Succeeded and success=true, see line 185 below